draft-ietf-quic-multipath.md

```diff
@@ -824,46 +829,30 @@ Path ID 1), and server provides two connection IDs
 Before the client opens a new path by sending a packet on that path
 with a PATH_CHALLENGE frame, it has to check whether there is
 an unused connection IDs for the same unused Path ID available for each side.
+In this example the Path ID 1 is used which is the smallest unused Path ID available
+as recommended in {{consume-retire-cid}}.
 Respectively, the client chooses the connection ID S1
 as the Destination Connection ID of the new path.
 
 
 ## Path Closure
 
-In this example, the client detects a network environment change
-(client's 4G/Wi-Fi is turned off, Wi-Fi signal is fading below a threshold,
-or the quality of RTT or loss rate is becoming worse) and wants to close
-an existing path.
-
-{{fig-example-path-close1}} illustrates an example of path closing. For the first path, the
-server's 1-RTT packets use DCID C1 and the
-client's 1-RTT packets use DCID S1 for the path with Path ID 1. For the
-second path, the server's 1-RTT packets use DCID C2 and
-the client's 1-RTT packets use DCID S2 for the path with Path ID
-2.
-
-Note that the paths use different packet number spaces. In this case, the
-client is going to close the first path with Path ID 1 but it sends the
-ABANBON frame over the second path using the DCID S2. In this example, the
-server confirms the path closure by sending an PATH_ABANDON frame
-by for the same Path ID to the client also using the other path with DCID C2.
-Both the client and the server can close the path after receiving
-the MP_RETIRE_CONNECTION_ID frame for that path.
+{{fig-example-path-close1}} illustrates an example of path closure.
+
+In this example, the client wants to close the path with Path ID 1.
+It sends the PATH_ABANDON frame to terminate the path. After receiving
+the PATH_ABANDON frame with Path ID 1, the server also send a
+PATH_ABANDON frame with Path ID 1.
 
 ~~~
 Client                                                      Server
 
-(client tells server to abandon a path)
-1-RTT[X]: DCID=S2 PATH_ABANDON[Path ID=1]->
+(client tells server to abandon a path with Path ID 1)
+1-RTT[X]: DCID=S1 PATH_ABANDON[Path ID=1]->
```

Comment:
But from the above CID negotiation, S1 belongs to Path ID 1. Should we advertise the PATH_ABANDON on the path we want to close? I do not think this is invalid, but... (see below)